### PR TITLE
Weather

### PR DESCRIPTION
**Pull request by 1v4n4**
This weather forecast site uses weather OpenWeatherMap.org API to get weather data for cities worldwide. By entering the name of the location, users get the current weather data.

Temperature is displayed in Celsius by default, but it can be changed to Fahrenheit (and back) by clicking on F/C letters next to the current temperature. 

The design is basic and responsive. There are different backgrounds for some weather conditions.

### CODE REVIEW

**Review by george-shammar — CHANGES_REQUESTED** [review]
### **STATUS: CHANGES REQUIRED** :recycle: :recycle: :construction: :construction_worker_man: 

@1v4n4 , Nice job and progress so far. :+1: 

**Highlights**
:heavy_check_mark: You have used the correct gitflow.
:heavy_check_mark: LInters is not informing of any errors.
:heavy_check_mark: Your project looks good already. Backgrounds are changing based on selected cities. 
:heavy_check_mark: You demonstrated understanding of async/await.

Great work :clap: 

Now, let's make it even better. Please see below required changes.

- [x] Please, implement a toggle control to determine if the result will be shown in Celsius or Fahrenheit. A user should be able to toggle betwen Celcius and Fahrenheit.
- [x] Include the `"dev": "webpack --mode development",`  scripts run command in your package.json.
- [ ] Your README should have instructions about where and how to generate any necessary keys, credentials, how to set, store, and generate them(if necessary).

Please make the required changes and submit another CRR when you're ready.

Kindly drop your message here and ping me on slack if you require clarification or supoort.

**Cheers** :smile **With respect** :tophat:

**Comment by 1v4n4** [comment]
Hi George, thanks for the quick review. I do have C/F toggle, it is activated by clicking on C or F. Other stuff I will gladly improve

File: package.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
+{
+  "name": "weather-app",
+  "version": "1.0.0",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "watch": "webpack --watch",
+    "start": "webpack serve --open",
+    "build": "webpack"
+  },
```
Comment (george-shammar) [inline package.json:10]: - [ ] Please include `    "dev": "webpack --mode development",` in your scripts.

**Review by george-shammar — CHANGES_REQUESTED** [review]
### **STATUS: CHANGES REQUIRED** :recycle: :recycle: 

@1v4n4 , Thank you for making some of the required changes. 

:heavy_check_mark: You have updated the README.
:heavy_check_mark: I can confirm that you've implemented the toggle feature.

**To Improve**

- [x] Kindly include   `    "dev": "webpack --mode development",` in your scripts. Please see my in-line comment below.

[OPTIONAL]

- [ ] For the toggle feature, you may want to specify in your README where it is. Right now, it is not easily spotted. Users (like myself) may simply see it as a unit of the temperature value.

Please make the required change and submit another CRR when you're ready.

Drop your message here and ping me on slack if you require clarifications or support.

Cheers :smile: With respect :tophat:

File: package.json (hunk 1 above)
Comment (1v4n4) [reply package.json]: Done!

**Review by BrianSammit — APPROVED** [review]
STATUS: APPROVED 🟢 💪 
--------
Hello @1v4n4, great job you did in this project 😄 👍 goo job working on the required changes 💯 
----------

- [x] [OPTIONAL] On your readme file on the `usage` section you highlight `npm run` but you missed the last part of the command to run the command properly `npm run build` and npm run start`. Please fix this 

--------------
Now you can merge this branch 🥇 👏 
HAPPY CODING!!! 😃 🍀 

giphy

**Pull request merged**